tests/pathfinding/test_blockchain_integration.py

```diff
@@ -108,30 +106,11 @@ def test_pfs_with_mocked_client(  # pylint: disable=too-many-arguments
     )
 
     # check that deposits, settle_timeout and transfers got registered
-    for (
-        index,
-        (
-            _p1_index,
-            p1_deposit,
-            _p1_capacity,
-            _p1_fee,
-            _p1_reveal_timeout,
-            _p1_reachability,
-            _p2_index,
-            p2_deposit,
-            _p2_capacity,
-            _p2_fee,
-            _p2_reveal_timeout,
-            _p2_reachability,
-            _settle_timeout,
-        ),
-    ) in enumerate(channel_descriptions_case_1):
+    for index, _ in enumerate(channel_descriptions_case_1):
```

Comment:
Nice to see this more concise. Since we don't need the list elements, we could even go with
```suggestion
    for index in range(len(channel_descriptions_case_1)):
```